RIOT-OS/RIOT
# Congure/feat/congure abe

### Contribution description
This provides an implementation for CongURE of the congestion control mechanism of TCP Alternative Backoff with ECN (ABE) (TCP ABE) as specified in [RFC 8511](https://tools.ietf.org/html/rfc8511).




### Testing procedure
Tests are provided for which [`riotctrl`](https://github.com/RIOT-OS/riotctrl/) (at least v0.2.2) is required for them to run. If installed,

```
make -j -C tests/congure_abe flash test
```

should succeed.



### Issues/PRs references
Requires #15951 and #15953. When merged, I will adapt the test framework to use the turo framework introduced in #15950, which was the original idea to begin with (which is why it is already printing JSON strings).

![PR dependency graph](https://page.mi.fu-berlin.de/mlenders/congure.svg)
